scenariolib/event_search.go

```diff
@@ -48,10 +47,26 @@ func newSearchEvent(e *JSONEvent, c *Config) (*SearchEvent, error) {
 			return nil, errors.New("Parameter custom must be a json object (map[string]interface{}) in a search event.")
 		}
 	}
+
+	if e.Arguments["matchLanguage"] != nil {
+		if matchLanguage, validCast = e.Arguments["matchLanguage"].(bool); !validCast {
+			return nil, errors.New("Parameter matchLanguage must be a type bool in SearchEvent")
+		}
+	} else {
+		matchLanguage = false
+	}
+
```

Comment:
well to be honnest the `matchLanguage = false` is not even needed because it's the default value of a bool.

I don't see a case where you would on purpose add `matchLanguage:false` in your scenarios.

But you are right it's not a wrong thing